readme.md

```diff
@@ -121,13 +141,21 @@ What is the difference between the two following lines of code?
 
 > Your answer...
 
+@artist.save! is being banged which modifies the actual value and provides extra errors if
+something goes wrong
```

Comment:
This is sort-of correct, but not very precise. `.save` and `.save!` don't modify the Ruby object, they both just try to save to the database. The difference is that `.save` returns false if it fails, `.save!` throws an exception (error) if it fails.